CppCoreGuidelines.md

```diff
@@ -10945,7 +10945,7 @@ The definition of `a2` is C but not C++ and is considered a security risk
 
 ##### Reason
 
-It nicely encapsulates local initialization, including cleaning up scratch variables needed only for the initialization, without needing to create a needless nonlocal yet nonreusable function. It also works for variables that should be `const` but only after some initialization work.
+It nicely encapsulates local initialization, including cleaning up scratch variables needed only for the initialization, without needing to create a needless non-local yet non-reusable function. It also works for variables that should be `const` but only after some initialization work.
```

Comment:
also a dictionary word, https://m-w.com/dictionary/nonreusable although I don't see it in the wild much (unlike "nonlocal")

Reply:
Thank you for the feedback!
Honestly I don't know which one is better here

#### nonresuable
* In the stdandard (N4842)
  * 0 times
* In dictionaries
  * Google : X
  * Oxford : X
  * Collins : O
  * Dictionary.com : O
  * Merriam-Webster : O
  * Macmillan : X
  * Cambridge : X
  * Longman : X

Reply:
I'll proceed with `non-reusable` for now
If you have any suggestion please let me know !